oc-api/config/functions.inc.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+function checkApiKey(){
+
+    if(!isset($_GET["oc-apikey"]) || $_GET["oc-apikey"] != getApiKey() || empty($_GET["oc-apikey"])){
+        echo UnauthorizedApiKey();
+        exit();
+    } else{
+        return true;
+    }
+}
+
+function UnauthorizedApiKey(){
+    
+    http_response_code(201);
```

Comment:
Set HTTP response status code to "401 Unauthorized".

```suggestion
    http_response_code(401);
```

Reply:
I forgot to reply to all of these comments. This has already been resolved, apparently I haven't pushed the updated version? Once I've worked on what I'm doing now, I'll push it with the resolved response code.